src/test/scala/HelloWorldSpec.scala

```diff
@@ -0,0 +1,53 @@
+import com.dimafeng.testcontainers.PostgreSQLContainer
+import com.example.{DatabaseInterface, User}
+import org.specs2._
+import zio.DefaultRuntime
+import zio._
+
+
+class HelloWorldSpec extends Specification {
+  def is = s2"""
+
+ This is my first test
```

Comment:
good that you have one end-to-end test, but we need to have separate tests for each method.